src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{20a2ba2c-5517-483f-8ffe-643441a59852}</ProjectGuid>
+    <ProjectGuid>{20A2BA2C-5517-483F-8FFE-643441A59852}</ProjectGuid>
```

Comment:
Yeah.  I started to undo it, and then decided to just leave it, so that we don't keep fighting with it :)